integreat_cms/static/src/js/media-management/component/edit-directory-sidebar.tsx

```diff
@@ -51,6 +51,31 @@ const EditDirectorySidebar = ({
         });
     };
 
+    const formatDate = (dateString: string): string => {
+        const months: { [key: string]: string } = {
+            "Jan.": "01",
+            "Feb.": "02",
+            "Mar.": "03",
+            "Apr.": "04",
+            "May": "05",
+            "June": "06",
+            "July": "07",
+            "Aug.": "08",
+            "Sept.": "09",
+            "Oct.": "10",
+            "Nov.": "11",
+            "Dec.": "12",
+        };
+
+        const [monthStr, day, yearStr] = dateString.split(" ");
+        const [year] = yearStr.split(",");
+        const month = months[monthStr];
+        const formattedDay = day.replace(",", "");
+
+        return `${formattedDay}.${month}.${year}`;
+    };
+    const formattedDate = formatDate(directory.CreatedDate.toLocaleString());
```

Comment:
I suggest using [date-fns](https://date-fns.org/) for all things related to dates as it is a proven utility library. It's less likely that we'll run into bugs compared to building it ourselves. :)

Reply:
@deen13 thank you for your review :) I think now there is no need in using library. It works without any additional formating